### DISCUSSION TITLE
Question: How to install Centreon without using the web-UI ?

### DISCUSSION THREAD

**Discussion by 51CGO** [centreon-archive/centreon-archived#10235] [Q&A] [answered]
This is a very simple question:

I need to setup Centreon _using scripts_. But once the RPM are installed, one must connect to the web-UI to complete the setup. I'd like to complete the setup wthout using the web-UI. Can you telle me which commands are called by the web-UI to:
1. Record configuration
2. initialize the database

Thank you.

**Comment by lgcosta** [comment, marked as answer]
Hi @crigoda ,

See about the `unattended.sh` script, it does everything you want.

-  https://docs.centreon.com/current/en/installation/introduction.html
-  https://github.com/centreon/centreon/blob/21.04.x/unattended.sh

**Reply by 51CGO** [reply]
You saved my life !
Thank you.
